### DISCUSSION TITLE
Create testbed to test rules, get rules prevalence, and find regressions

### DISCUSSION THREAD

**Discussion by mr-tz** [mandiant/capa#2002] [Ideas]
Let's discuss the creation of a testbed of samples (in addition/separate from the `tests/data`) to:
- test (new) rules
- obtain an approximation of rule hit prevalence
- find regressions

Some rough tasks:
- sketch out desired functionality and requirements
- collect 1000 - 10,000 samples (benign and malicious)
- create and automate `frz` file (frozen features) or BinExport creation to speed up processing
- create scripts and automation to facilitate the desired use-cases

CC: @Aayush-Goel-04 who helped to come up with this idea

**Comment by Aayush-Goel-04** [comment]
Seems like issue #424 is relevant to this discussion. I'll link it here for reference.
